src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -90,6 +90,9 @@ public static class MachineOptions {
                 "1.0 means 1L Steam -> 1 EU. 0.5 means 2L Steam -> 1 EU.", "Default: 0.5" })
         public double multiblockSteamToEU = 0.5;
 
+        @Config.Comment({ "Nuclear Max Power multiplier for balancing purposes", "Default: 2.0" })
```

Comment:
what context is this value used in?

Reply:
This is basically used to calculate the maximum amount of hot coolant that could possibly be exiting the reactor per second. If this is increased, that amount is also increased for any reactor design. If a pack developer wants to nerf nuclear, this is one of their primary levers.